src/command_modules/azure-cli-network/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "2.0.18"
+VERSION = "2.0.19"
```

Comment:
Should helpfile-only changes bump a version? I personally don't think so but this is for @JasonRShaver to decide.

Reply:
This is required as 2.0.18 is currently available (https://pypi.python.org/pypi/azure-cli-network).
If it was kept at 2.0.18, the changes would not be released as 2.0.18 is already available.

Reply:
Addressed.